prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -981,27 +1012,6 @@ In order to pull AMP parameters out into targeting, Prebid Server places AMP que
 }
 ```
 
-##### EID Permissions (PBS-Go only)
-
-This feature allows publishers to specify ext.prebid.eidpermissions, defining which extended ID
-in user.ext.eids is allowed to be passed to which bid adapter. For example:
-
-```
-{
-  "ext": {
-    "prebid": {
-      "data": {
-        "eidpermissions": [   // prebid server will use this to filter user.ext.eids
-          {"source": "sharedid.org", "bidders": ["*"]},  // * is the default
-          {"source": "neustar.biz",  "bidders": ["bidderB"]},
-          {"source": "id5-sync.com", "bidders": ["bidderA","bidderC"]}
-        ]
-      }
-    }
-  }
-}
-```
-
```

Comment:
Why is this removed? Did you intend to just remove the PBS-Go only label?

Reply:
That eidspermission section was duplicated. Maybe it was there for PBS-Go only at first, then when PBS-Java implemented it I re-documented and didn't notice the existing section?